### PR TITLE
Gameplay hint for shades

### PR DESCRIPTION
**Pull request by Pathid**
Seeing how quick as these guys tend to die, and as it's not often a player gets "drafted" into playing one, I figure it's fair to give them a little advice as to how they work. There seems to be a common misconception that shades heal by attacking living beings.

### CODE REVIEW

**Comment by Aurx** [comment]
This doesn't really clarify that misconception.  
And it's an entirely reasonable misconception given their attack text.

**Comment by PublicEnemyMaiPai** [comment]
Why not change attack text or even let them steal life?

**Comment by Pathid** [comment]
Right. I should change that attack text at least.

**Comment by Pathid** [comment]
If it's too wordy I can pare it down some.

**Comment by DamianX** [comment]
I'm more concerned about the usage of the "afflicts" verb here. Something like "torments" seems more fitting/correct to me.

**Comment by Pathid** [comment]
"Torments" is also good.
